core/themes/seven/css/style.css

```diff
@@ -1160,6 +1160,27 @@ textarea.form-textarea:focus {
   color: #000;
   border-color: #43afe4;
 }
+input[type=color] {
```

Comment:
All other selectors here use a class name. Can `input.form-color` be used instead?